Tests/PureeTests/Output/BufferedOutputDispatchQueueTests.swift

```diff
@@ -0,0 +1,37 @@
+import Foundation
+import XCTest
+@testable import Puree
+
+private func makeLog() -> LogEntry {
+    return LogEntry(tag: "foo", date: Date())
+}
+
+class BufferedOutputDispatchQueueTests: XCTestCase {
+
+    func testFlushInterval() {
+        let exp = expectation(description: #function)
+
+        // Run the test on a background queue
+        let dispatchQueue = DispatchQueue(label: "com.cookpad.Puree.Logger", qos: .background)
+        dispatchQueue.async {
+
+            // Setup the output
+            let logStore = InMemoryLogStore()
+            let output = TestingBufferedOutput(logStore: logStore, tagPattern: TagPattern(string: "pv")!, options: nil)
```

Comment:
`TestingBufferedOutput` is not real implementation. it is just a mock.
So this test seems to be unreliable.

Reply:
What needs to be confirmed here is just that `write` is called within an expected amount of time. I guess this mock is just enough for that purpose.